community/governance.md.tmpl

```diff
@@ -37,14 +37,24 @@ Here are defined the primary teams participating in conda-forge activities.
   core, part of staged-recipes, or maintainers. This includes first-time
   contributors, collaborators, and funders. They have no special rights within
   the conda-forge organization itself.
-* **emeritus-core:** Core members that are inactive (commits, GitHub comments/issues/reviews,
-  dev meetings and voting on polls) in the past six months will be asked if they want to become emeritus-core
-  developers. Any core member can also request to become emeritus if they wish to do so
-  (e.g. taking a sabbatical or long vacation).
-  Emeritus core members can still vote and move back to active core anytime. Emeritus
-  votes are used to count towards quorum but the quorum size is computed from the size of
-  the active core group. The `core.csv` list should be updated when a change in the status
-  of a member occurs.
+* **emeritus-core:** Core members that do not show any activity in their core
+  responsibilities (including organization maintenance in the form of commits,
+  GitHub comments, issues, or reviews; participation in chat platforms and
+  community meetings; voting on polls) in the past six months will be asked if
+  they want to become emeritus-core developers. Four weeks after asking, if the
+  inactive core member has not responded, they will be automatically moved to
+  emeritus-core. Every 12 months or more, one core member should ask the rest of
```

Comment:
"Every 12 months or more" is IMO not a great formulation for a governance document. If I plan to ask once every 100 years, that would still be covered by the "or more" part.

I would just make it 12 month; it's already qualified with "should" anyway, so not like it's binding.

Reply:
I was trying to prevent strict 12-month windows, but on second thought it's true it doesn't make a lot of sense. I'll rephrase.

Reply:
```suggestion
  emeritus-core. Every 12 months, one core member should ask the rest of
```